### PR TITLE
Create random-even-number.java

### LINKED ISSUE #1110: Write a Java programme to generate a random even number in range n1-n2
### Description

Write a programme to generate a random even number in range n1-n2

```
Start range : 0
End   range : 100

Output : 44
```

### PR DESCRIPTION
**Pull request by SpirosArk**
### Why:

Closes #1110 

### Check off the following:

- [x ] I have reviewed my changes in staging (look for "Automatically generated comment" and click **Modified** to view your latest changes).
- [x ] For content changes, I have completed the [self-review checklist](https://github.com/codinasion/codinasion/blob/master/contributing/self-review.md).

### CODE REVIEW

File: programme/random-even-number/random-even-number.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
+//Program to generate random even number in range using Java
```
Comment (KikiManjaro) [inline programme/random-even-number/random-even-number.java:1]:
```suggestion
//Programme to generate random even number in range using Java
```

For consistency we are now using `programme` everywhere and not `program`

File: programme/random-even-number/random-even-number.java [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+        System.out.println("Enter end range:");
+        int endRange = s_name.nextInt();
+        
+        System.out.println("Random value in int from "+startRange+" to "+endRange+ ":\n");
+        int random_int = (int)Math.floor(Math.random()*(endRange-startRange+1)+startRange);
+        while (random_int % 2 != 0){
+            
+            random_int = (int)Math.floor(Math.random()*(endRange-startRange+1)+startRange);
+            
+        }
```
Comment (KikiManjaro) [inline programme/random-even-number/random-even-number.java:21]:
```suggestion
        while (random_int % 2 != 0){
            random_int = (int)Math.floor(Math.random()*(endRange-startRange+1)+startRange);
        }
```

You don't need those empty space here

File: programme/random-even-number/random-even-number.java [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    public static void main(String[] args) {
+        Scanner s_name = new Scanner(System.in);
+        
+        System.out.println("Enter start range:\n");
+        int startRange = s_name.nextInt();
+        System.out.println("Enter end range:");
+        int endRange = s_name.nextInt();
+        
+        System.out.println("Random value in int from "+startRange+" to "+endRange+ ":\n");
+        int random_int = (int)Math.floor(Math.random()*(endRange-startRange+1)+startRange);
```
Comment (KikiManjaro) [inline programme/random-even-number/random-even-number.java:16]:
```suggestion
        System.out.println("Random value in int from " + startRange + " to " + endRange + ":\n");
        int random_int = (int) Math.floor(Math.random() * (endRange - startRange + 1) + startRange);
```

In Java you should add space between operators

**Review by harshraj8843 — APPROVED** [review]
👍🏻

**Pull request merged**